analyzer/src/analysis/value/interprocedural/sequential/function_fixpoint.cpp

```diff
@@ -76,7 +77,17 @@ FunctionFixpoint::FunctionFixpoint(
       _checkers(checkers),
       _exit_invariant(make_bottom_abstract_value(ctx)),
       _return_stmt(nullptr),
-      _logger(logger) {}
+      _logger(logger)
+{
+  if (_ctx.opts.trace_ar_statements) {
+    ar::TextFormatter formatter{};
+    auto& stream = analyzer::log::msg().stream();
+    stream << "\n>>>>>>>>>>>>>>\nEntering Interprocedural Sequential FunctionFixpoint for ";
+    ar::Namer namer;
```

Comment:
I think we need to give it the `ar::Code*` so it properly names local variables.
Also, what about storing it as an attribute so it's only done once? I could be a `std::unique_ptr<ar::Namer>`.